networking/network_observability/installing-operators.adoc

```diff
@@ -1,13 +1,11 @@
 :_content-type: ASSEMBLY
 [id="installing-network-observability-operators"]
-= Installing Network Observability operators in {product-title}
+= Installing the Network Observability operator
 include::_attributes/common-attributes.adoc[]
 :context: network_observability
 
 toc::[]
+The Loki Operator integrates with the Network Observability operator to implement multi-tenancy & authentication. The Network Observability operator is not multi-tenant. Installing the Loki operator is a  prerequisite for using the Network Observability operator. You can install the the Loki Operator using the {product-title} web console Operator Hub.
```

Comment:
```suggestion
The Loki Operator integrates with the Network Observability operator to implement authentication. Installing the Loki operator is a prerequisite for using the Network Observability operator. You can install the the Loki Operator using the {product-title} web console Operator Hub. The Network Observability operator is not multi-tenant. You must provide a dedicated LokiStack instance for it.
```

We should mention to avoid sharing Loki Operator instances (nammed `LokiStack` in their operator)